### PR TITLE
Add OAI admin emails setting, refs #8156

### PR DESCRIPTION
**Pull request by jraddaoui**
- Add migration and fixtures for new setting
- Add field to OAI settings form
- Use setting value in OAI identify component

### CODE REVIEW

File: data/fixtures/settings.yml [hunk 1]
```diff
@@ -515,6 +515,13 @@ QubitSetting:
     editable: 1
     deleteable: 0
     value: 100
+  QubitSetting_oai_repository_code:
```
Comment (mcantelon) [inline data/fixtures/settings.yml]: You might have forgot to change this key in the YAML.

File: plugins/arOaiPlugin/modules/arOaiPlugin/templates/_identify.xml.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <baseURL><?php echo QubitOai::getBaseUrl() ?></baseURL>
     <protocolVersion><?php echo $protocolVersion ?></protocolVersion>
     <?php
-    foreach($adminEmail as $email){
+    foreach($adminEmails as $email){
```
Comment (mcantelon) [inline plugins/arOaiPlugin/modules/arOaiPlugin/templates/_identify.xml.php]: Minor style nitpick: the "{" should be moved to the following line (I probably put the incorrect code style in there originally).

Comment (mcantelon) [inline plugins/arOaiPlugin/modules/arOaiPlugin/templates/_identify.xml.php]: (I think there's supposed to be a space between "foreach" and "(" too.)

**Comment by jraddaoui** [comment]
Thanks Mike! Merged.

**Pull request closed without merging**